packages/sdk-react/src/components/receivables/InvoiceDetails/CreateReceivable/CreateReceivables.tsx

```diff
@@ -203,6 +204,7 @@ const CreateReceivablesBase = (props: InvoiceDetailsCreateProps) => {
                   onCurrencyChanged={setActualCurrency}
                 />
                 <PaymentSection disabled={createReceivable.isPending} />
+                <ReminderSection disabled={createReceivable.isPending} />
```

Comment:
We need the same section for the invoice edit form too.

Reply:
Can you please provide at which component ?